### DISCUSSION TITLE
Moving a chart within the reports tab not working as expected.

### DISCUSSION THREAD

**Discussion by isley-griffith** [supabase/supabase#16928] [Contribute to Supabase]
# Bug report

## Describe the bug
When I add multiple report charts and then attempt to reorder them, they do not swap order correctly, they mostly push each other around without snapping back into place.

## To Reproduce

1. Go to the Reports tab
2. Add some charts (I have 8 open)
3. Drag them around
4. Notice how they do not snap back to a grid

## Expected behavior

1. Add charts
2. Drag them around
3. Maintain a grid ordering

## Screenshots

![image](https://user-images.githubusercontent.com/48958109/219463060-d90403a4-e566-4055-92bb-abd0428fb167.png)
![image](https://user-images.githubusercontent.com/48958109/219463174-2e8485b1-7d83-4c7a-a1b3-49ad7983d4c9.png)

## System information

- OS: macOS
- Browser: Chrome

## Additional context

Perhaps this is the intended behavior. If so, then please disregard.

**Comment by MildTomato** [comment]
The reports page pretty much uses [react-grid-layout](https://github.com/react-grid-layout/react-grid-layout) with pretty default config I think.
Maybe there's some config we can change, it looks like we implemented it the ["messy" way](https://react-grid-layout.github.io/react-grid-layout/examples/3-messy.html), but the default behavior is perhaps to push all the items to the top.
